shacl/DCAT-AP-NO-shacl_shapes_2.00.ttl

```diff
@@ -1029,16 +1029,17 @@
         sh:path skos:inScheme
     ] .
 
-:CorporateBodyRestriction
-    a sh:NodeShape ;
-    rdfs:comment "Corporate Body Restriction" ;
-    rdfs:label "Corporate Body Restriction" ;
-    sh:property [
-        sh:hasValue <http://publications.europa.eu/resource/authority/corporate-body> ;
-        sh:minCount 1 ;
-        sh:nodeKind sh:IRI ;
-        sh:path skos:inScheme
-    ] .
+#### this one is much to strict to require, as national org. are not on the list
```

Comment:
Kan du vurdere å fjerne fullstending (delete) i staden for å kommentere ut? Blir mykje støy. 
Evt kunne vi vurdere å dokumentere avvik frå EU i feks README.md i staden for inne i shacl-fila

Reply:
Kan godt gjøre det (smak og behag, jeg liker å ha dokumentasjon i koden, når det er mulig) - kommer ny commit snart.

Reply:
... så vidt jeg husker har jeg forresten lagt inn flere slike kommentarer (på avvik), rydder senere

Reply:
Fint.